connectors/v2/wdvx.js

```diff
@@ -2,16 +2,12 @@
 
 /* global Connector */
 
-Connector.playerSelector = '#wrap div.fixed';
+Connector.playerSelector = '.audiojs';
 
-Connector.getArtist = function () {
-	return $('.artist').text().replace('Artist: ','');
-};
+Connector.artistSelector = '#now-playing .artist';
 
-Connector.getTrack = function () {
-	return $('.title').text().replace('Title: ', '');
-};
+Connector.trackSelector = '#now-playing .title';
 
-Connector.isPlaying = function () {
-	return $('.pauseW').is(':visible');
+Connector.isPlaying = function() {
+	return $('.audiojs').hasClass('playingW');
```

Comment:
No, it's not a typo:
![image](https://cloud.githubusercontent.com/assets/1119267/19478139/f757e432-9549-11e6-82f3-ae6111a6db3b.png)